src/com/tekwill/topic5/Patrat.java

```diff
@@ -0,0 +1,26 @@
+package com.tekwill.topic5;
+
+public class Patrat extends FiguraGeometrica {
+
+    private double lungimeaLaturaPtrat;
+
+    @Override
+    double returneazaAria() {
+        double aria = lungimeaLaturaPtrat * lungimeaLaturaPtrat;
```

Comment:
aici ai creat variabila local aria. Cred ca am fi putut crea o variabila de instanta cu numele aria, chiar in clasa Parinte, având in vedere ca aria este o chestie comuna pentru absolut toate figurile geometrice.